frontend/src/components/Pages/FrontPage/MissionOverview/StopMissionDialog.tsx

```diff
@@ -0,0 +1,100 @@
+import { Button, Dialog, Typography, Icon } from '@equinor/eds-core-react'
+import styled from 'styled-components'
+import { TranslateText } from 'components/Contexts/LanguageContext'
+import { Icons } from 'utils/icons'
+import { useState, useEffect } from 'react'
+import { tokens } from '@equinor/eds-tokens'
+import { Mission } from 'models/Mission'
+import { useMissionControlContext } from 'components/Contexts/MissionControlContext'
+
+const StyledDisplayButtons = styled.div`
+    display: flex;
+    width: 410px;
+    flex-direction: columns;
+    justify-content: flex-end;
+    gap: 0.5rem;
+`
+
+const StyledDialog = styled(Dialog)`
+    display: grid;
+    width: 450px;
+`
+
+const StyledText = styled.div`
+    display: grid;
+    gird-template-rows: auto, auto;
+    gap: 1rem;
+`
+
+interface MissionProps {
+    mission: Mission
+}
+
+export enum ControlButton {
+    Pause,
+    Stop,
+    Resume,
+}
```

Comment:
Duplicated in both StopMissionPopOver.tsx and MissionControlButtons.tsx. Should only be in one of them and imported to the other

Reply:
I have changed this and the enum is exported on StopMissiondialog.tsx